charts/newrelic-infrastructure/templates/configmap.yaml

```diff
@@ -18,6 +17,8 @@ metadata:
   labels: {{ include "newrelic.labels" . | indent 4 }}
   name: {{ template "newrelic.fullname" . }}-integrations-cfg
 data:
+{{ (.Files.Glob "flex/**.yaml").AsConfig | indent 2 }} # include Flex configs
```

Comment:
I think this is a clever approach, but I'm not 100% sure we need a generic way of including files if we don't foresee needing more, especially since users will not be able to add files to this folder.

Reply:
@roobre I made this more specific [here](https://github.com/newrelic/helm-charts/pull/707/files#diff-a6b2b5ef195e7a5534a265575a2bfdc196c15e03b15f4d27af7ffb1e76bbf413R20-R23).